source/includes/include-fixes/mongodb-compatibility-table-go.rst

```diff
@@ -0,0 +1,37 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Go Driver Version
+     - MongoDB 8.0
+     - MongoDB 7.0
+     - MongoDB 6.1
+     - MongoDB 6.0
+     - MongoDB 5.0
+     - MongoDB 4.4
+     - MongoDB 4.2
+     - MongoDB 4.0
+     - MongoDB 3.6
+
+   * - 1.12 to 1.17
+     - ⊛ [#8.0-support]_
```

Comment:
Looks like we're missing the footnote for this bullet. Can you add it? You can see it in upcoming here: https://www.mongodb.com/docs/drivers/go/upcoming/compatibility/#ref-8.0-support-id1

Reply:
Ah thanks will add! It was commented out in the sharedinclude so I missed it.